src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -592,12 +572,13 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
     }
 
     /* All empty lines or lines with only white spaces will be skipped.
-     * The invalidity info returned are in HTML format.
+     * The invalidity info is in HTML format.
      */
-    private List<String> getInvalidityInfoInEnrollLines(String lines, String courseId) throws EnrollException {
+    public List<StudentAttributes> createAndValidateStudents(String lines, String courseId) throws EnrollException {
```

Comment:
I don't think `Validate` should be in the method name. Methods are supposed to validate data anyway.